src/stores/OrganisationStore.ts

```diff
@@ -2,17 +2,24 @@ import { defineStore, type Store, type StoreDefinition } from 'pinia';
 import { isAxiosError, type AxiosResponse } from 'axios';
 import {
   OrganisationenApiFactory,
-  OrganisationResponseTypEnum,
-  CreateOrganisationBodyParamsTypEnum,
-  CreateOrganisationBodyParamsTraegerschaftEnum,
+  OrganisationsTyp,
+  TraegerschaftTyp,
   type OrganisationenApiInterface,
   type CreateOrganisationBodyParams,
-  type OrganisationResponseLegacy,
 } from '../api-client/generated/api';
 import axiosApiInstance from '@/services/ApiService';
 
 const organisationApi: OrganisationenApiInterface = OrganisationenApiFactory(undefined, '', axiosApiInstance);
 
+export type Organisation = {
```

Comment:
Wieso nehmen wir hier nicht den Typ, der von der API kommt?

Reply:
Ich denke, es geht darum, uns nicht vollständig darauf zu verlassen und Probleme zu vermeiden, wie wir sie mit der neuen legacy Organisationresponse hatten.

Reply:
Verstehe den Grund, aber so müssten wir bei jeder Erweiterung in der Api auch den Typ im Store anpassen, anstatt den automatisch von der Api zu übernehmen. Würde ich verstehen, wenn wir Änderungen am Typ haben, aber so ist es ja 1:1 der gleiche Typ.

Reply:
Stellt ihr dafür einen Termin ein? Dann ist das hier erstmal resolved